lib/OpenAPI/Client.pm

```diff
@@ -19,8 +19,19 @@ has base_url => sub {
   my $schema  = $self->validator->schema;
   my $schemes = $schema->get('/schemes') || [];
 
-  return Mojo::URL->new->host_port($schema->get('/host') || 'localhost')->path($schema->get('/basePath') || '/')
-    ->scheme($schemes->[0] || 'http');
+  my $url;
+
+  # Schema loaded is an OpenAPIv3 spec
+  if ($schema->get('/openapi')) {
+    $url = Mojo::URL->new($schema->get('/servers')->[0]{url});
```

Comment:
What happens if there are no servers defined?

Reply:
You are right, according to the spec, servers could be omitted from the spec or be an empty array. If this is the case, it should default to '/'.

Addressing this in the next push